playground/react/App.tsx

```diff
@@ -1,14 +1,22 @@
-import { StoryblokComponent, useStoryblok } from '@storyblok/react';
+import { StoryblokRichText, useStoryblok } from '@storyblok/react';
 import React from 'react';
 
 function App() {
-  const story = useStoryblok('home', { version: 'draft' });
+  // const story = useStoryblok('home', { version: 'draft' });
+  const story = useStoryblok('react/test-richtext', { version: 'draft' });
 
   if (!story?.content) {
     return <div>Loading...</div>;
   }
 
-  return <StoryblokComponent blok={story.content} />;
+  /* return <StoryblokComponent blok={story.content} />; */
+  return (
+    <div>
+      <StoryblokRichText doc={story.content.richText} />
+      {/* <StoryblokComponent blok={story.content} />
+      ; */}
```

Comment:
same, if we want to leave the RichText component here, we should keep also the original Component or move it into another page